packages/devtools_app/lib/src/flutter/split.dart

```diff
@@ -46,11 +71,37 @@ class Split extends StatefulWidget {
   final Axis axis;
 
   /// The child that will be laid out first along [axis].
+  ///
+  /// If null, [firstBuilder] will be used instead.
+  ///
+  /// Between [firstBuilder] and [firstChild], exactly one must be non-null.
+  /// In other words, `(firstBuilder == null) != (firstChild == null)`.
   final Widget firstChild;
 
   /// The child that will be laid out last along [axis].
```

Comment:
nit: r/last/second to match builder comment

Reply:
Done, thanks for catching this!